### PR TITLE
docs: fix broken link redis

### PR DESCRIPTION
**Pull request by Keshraf**
Fixes missing link on this page: https://docs.medusajs.com/advanced/backend/notification/how-to-create-notification-provider#prerequisites

Please add a Hacktober label to this pull request 🙌

### CODE REVIEW

File: docs/content/advanced/backend/notification/how-to-create-notification-provider.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ If you’re unfamiliar with the Notification architecture in Medusa, it is recom
 
 Before you start creating a Notification Provider, you need to install a [Medusa server](../../../quickstart/quick-start.md).
 
-You also need to (../../../tutorial/0-set-up-your-development-environment.mdx#redis) and [configure it with the Medusa server](../../../usage/configurations.md#redis).
+You also need to [setup redis](../../../tutorial/0-set-up-your-development-environment.mdx#redis) and [configure it with the Medusa server](../../../usage/configurations.md#redis).
```
Comment (shahednasser) [inline docs/content/advanced/backend/notification/how-to-create-notification-provider.md:15]:
```suggestion
You also need to [setup Redis](../../../tutorial/0-set-up-your-development-environment.mdx#redis) and [configure it with the Medusa server](../../../usage/configurations.md#redis).
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for noticing this and opening a PR! Just a small change below

File: docs/content/advanced/backend/notification/how-to-create-notification-provider.md (hunk 1 above)
Comment (Keshraf) [reply docs/content/advanced/backend/notification/how-to-create-notification-provider.md:15]: Makes sense to Capitalise.

**Comment by Keshraf** [comment]
> Thank you for noticing this and opening a PR! Just a small change below

@shahednasser Done!

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Awesome thanks! And since the PR was approved and merged, there's no need for the label 😊

**Pull request merged**